app/src/main/java/blockeq/com/stellarwallet/utils/StringFormat.kt

```diff
@@ -40,5 +42,18 @@ class StringFormat {
             }
             return false
         }
+
+        fun getAssetCode() : String {
```

Comment:
This seems to be a very app specific function and doesn't need string formatting - we can take it out of the utilities and probably more suitable in a more session object helper (or something similar)